reference/tiflash/scale.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Scale the TiFlash Cluster
+summary: Learn how to scale in and out nodes in the TiFlash cluster.
+category: reference
+---
+
+# Scale the TiFlash Cluster
+
+This document describes how to scale in and out nodes in the TiFlash cluster.
+
+## Scale out a TiFlash node
+
+The following example shows how to scale out a TiFlash node if you deploy TiFlash on the `192.168.1.1` node.
+
+1. Edit the `inventory.ini` file to add the TiFlash node information:
+
+    Currently, you can only add IP but not domain name.
+
+    {{< copyable "" >}}
+
+    ```ini
+    [tiflash_servers]
+    192.168.1.1
+    ```
+
+2. Edit the `hosts.ini` file to add the node information:
+
+    {{< copyable "" >}}
+
+    ```ini
+    [servers]
+    192.168.1.1
+    [all:vars]
+    username = tidb
+    ntp_server = pool.ntp.org
+    ```
+
+3. Initialize the new node:
+
+    - Configure the SSH mutual trust and sudo rules on the Control Machine:
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        ansible-playbook -i hosts.ini create_users.yml -l 192.168.1.1 -u root -k
+        ```
+
+    - Install the NTP service on the target machine:
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        ansible-playbook -i hosts.ini deploy_ntp.yml -u tidb -b
+        ```
+
+    - Initialize the node on the target machine:
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        ansible-playbook bootstrap.yml -l 192.168.1.1
+        ```
+    
+4. Deploy the new node:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    ansible-playbook deploy.yml -l 192.168.1.1
+    ```
+
+5. Start the new node:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    ansible-playbook start.yml -l 192.168.1.1
+    ```
+
+6. Update the configuration of Prometheus and restart:
```

Comment:
Please check with the author @ilovesoup `restart` what.

Reply:
Confirmed: restart `Prometheus`